centinel/experiments/baseline.py

```diff
@@ -101,10 +94,17 @@ def run_file(self, input_file):
                     file_comments.append(row)
                 continue
 
+            # detect the header row and store it
```

Comment:
Also, I don't think your link contains comments. You should include a 5-10 line example of what the file could look like with both comments (since they have a different format) and URLs